src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,216 @@
+package kitchenpos.application;
+
+import kitchenpos.domain.*;
+import kitchenpos.infra.PurgomalumClient;
+import kitchenpos.testfixture.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+    MenuRepository menuRepository = new InMemoryMenuRepository();
+
+    MenuGroupRepository menuGroupRepository = new InMemoryMenuGroupRepository();
+
+
+    ProductRepository productRepository = new InMemoryProductRepository();
+
+    PurgomalumClient purgomalumClient = new FakePurgomalumClient();
+
+    MenuService menuService;
+
+    @BeforeEach
+    void setUp() {
+        menuService = new MenuService(menuRepository, menuGroupRepository, productRepository, purgomalumClient);
+    }
```

Comment:
아래와 같이 변경이 필요할 것 같아요. 🙂
```suggestion
    private MenuRepository menuRepository;

    private MenuGroupRepository menuGroupRepository;

    private ProductRepository productRepository;

    private PurgomalumClient purgomalumClient;

    private MenuService menuService;

    @BeforeEach
    void setUp() {
        menuRepository = new InMemoryMenuRepository();
        menuGroupRepository = new InMemoryMenuGroupRepository();
        productRepository = new InMemoryProductRepository();
        purgomalumClient = new FakePurgomalumClient();

        menuService = new MenuService(menuRepository, menuGroupRepository, productRepository, purgomalumClient);
    }
```

--------------

이 부분은 확인해보니 이런식으로 꼭 변경하지 않아도 제가 생각한 것처럼 매 테스트 케이스에서 인스턴스를 새롭게 생성해서 주입해주고있군요? 🤔 

꼭 이렇게 반영하지는 않으셔도되지만, 명시적으로 새로운 인스턴스를 매번 할당한다는 느낌을 줬다 정도로 이해해주시면 될 것 같아요. 😅